crates/net/eth-wire/src/hello.rs

```diff
@@ -99,8 +99,9 @@ impl HelloMessageBuilder {
         HelloMessage {
             protocol_version: protocol_version.unwrap_or_default(),
             client_version: client_version.unwrap_or_else(|| DEFAULT_CLIENT_VERSION.to_string()),
-            capabilities: capabilities
-                .unwrap_or_else(|| vec![EthVersion::Eth67.into(), EthVersion::Eth66.into()]),
+            capabilities: capabilities.unwrap_or_else(|| {
+                vec![EthVersion::Eth68.into(), EthVersion::Eth67.into(), EthVersion::Eth66.into()]
+            }),
```

Comment:
what has changed since [this comment](https://github.com/paradigmxyz/reth/pull/1361#discussion_r1106925368)?

Reply:
The devp2p hive tests expect eth/68 in the Status, which at the time I'm not sure we knew due to other errors in the hive tests. So we need to advertise 68 now